backend/handlers/handlers.go

```diff
@@ -631,54 +631,78 @@ func (handlers *Handlers) getRatesHandler(_ *http.Request) (interface{}, error)
 func (handlers *Handlers) getConvertToFiatHandler(r *http.Request) (interface{}, error) {
 	from := r.URL.Query().Get("from")
 	to := r.URL.Query().Get("to")
-	amount := r.URL.Query().Get("amount")
-	amountAsFloat, err := strconv.ParseFloat(amount, 64)
-	if err != nil {
+	coin := r.URL.Query().Get("amount")
```

Comment:
`amount` is a better variable name? coin usually refers to a `coin.Coin` instance.